pkgs/development/libraries/libwebsockets/default.nix

```
@@ -34,6 +36,11 @@ stdenv.mkDerivation rec {
   ++ lib.optional withExternalPoll "-DLWS_WITH_EXTERNAL_POLL=ON";
 
   postInstall = ''
+    # Fix path that will be incorrect on move to "dev" output.
+    substituteInPlace \
+      "$out/lib/cmake/libwebsockets/LibwebsocketsTargets-release.cmake" \
+      --replace "\''${_IMPORT_PREFIX}" "$out"
```

```suggestion
    substituteInPlace "$out/lib/cmake/libwebsockets/LibwebsocketsTargets-release.cmake" \
      --replace "\''${_IMPORT_PREFIX}" "$out"
```

Also we should rather fix the cmake file to use the correct variable.

---

I allied the formatting change but I'm not very familiar with cmake so I'm not certain how I would fix it to use the correct variable 😑